### PR TITLE
add Yokogawa to list of Commercial Partners

### PR DESCRIPTION
**Pull request by lunson**
This is per Jason's direct request. Please do not merge until he gives me a thumbs up; I just need it staged so that he has a link to review when he's ready for the go-ahead. Thanks!

### CODE REVIEW

File: commercial-partners/index.html [hunk 1]
```diff
@@ -63,6 +63,7 @@ <h5>Commercial licenses of OME software</h5>
                     <li><a href="https://www.imagescience.de/imagic.html" target="_blank">Imagic, GmBH</a></li>
                     <li><a href="http://www.perkinelmer.com/" target="_blank">PerkinElmer</a></li>                
                     <li><a href="http://www.mayachitra.com/" target="_blank">Mayachitra Inc.</a></li>
+                    <li><a href=" https://www.yokogawa.com/solutions/products-platforms/life-science/" target="_blank">Yokogawa</a></li>
```
Comment (hflynn) [inline commercial-partners/index.html]: I assume this space between " and the URL is why Travis is complaining about this

**Comment by lunson** [comment]
Anyone know how to fix the errors?

**Comment by lunson** [comment]
@joshmoore @hflynn Thank you!

**Comment by hflynn** [comment]
Now staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/commercial-partners/ ready for Jason

**Comment by lunson** [comment]
This is okay to go live now. Jason might want the link changed in the future, but this is a go. Thank you!

**Pull request merged**